src/styles/blocks/card.scss

```diff
@@ -0,0 +1,124 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+
+  &__image{
+    width: 160px;
+    height: 134px;
+    margin-top: 32px;
+    margin-left: 19px;
+    margin-right: 19px;
```

Comment:
Remove it everywhere and add padding for the card instead of margins or paddings left and right